zerver/webhooks/dropbox/view.py

```diff
@@ -1,18 +1,85 @@
-from typing import Text
+from typing import Text, Any, List
 from django.http import HttpRequest, HttpResponse
 from zerver.lib.actions import check_send_stream_message
 from zerver.lib.response import json_success
 from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
 from zerver.models import UserProfile
+from dropbox import dropbox
+
+# POSSIBLE UPGRADE : MESSAGE WHEN A FOLDER/FILE IS SHARED TO WHICH USER
+#                  : Who updated the file/folder
+
+# Save past cursors of each user
+cursors = dict()
```

Comment:
We need to store past cursors for each dropbox access_code. How do you want me to implement it?
Currently it stores email of the user who created the webhook as key.

Reply:
We will have to store it in a file, because a server restart will delete all the past stored cursors.